packages/components/package.json

```diff
@@ -51,13 +51,13 @@
     "@types/vfile-message": "^2.0.0",
     "@typescript-eslint/eslint-plugin": "^2.20.0",
     "@typescript-eslint/parser": "^2.20.0",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.2",
     "babel-loader": "^8.0.6",
     "classnames": "^2.2.6",
     "copyfiles": "^2.2.0",
     "css-loader": "^3.4.2",
     "css-modules-typescript-loader": "^4.0.0",
     "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.2",
```

Comment:
test cases was breaking due to react-16 adapter so we needed adapter for version 17 and that is not available in market as this is perfectly new version.

so i just used this suggestion and added temporary package which will solve our issues after upgrades and later on whenever exact package release officially then we just need to update it.

https://stackoverflow.com/a/64597067/6536418

Reply:
Thanks @sanjayradadiya 

Could you add a comment in `package.json` next to the dependency, including a link to the Stack Overflow issue please?  Could you also create a GitHub issue in this repo explaining that we need to switch back to the proper adapter?

Reply:
Hello @sanjayradadiya - I'm keeping this PR open as a reminder that we need the above comment/documentation sorted.

Once you've done it I can merge the PR :-)

Reply:
This PR still needs some attention from you, please @sanjayradadiya

Reply:
Ahh, okay

Reply:
Please check this issue documentation here for sure

https://github.com/serge-web/serge/issues/950

Reply:
Nicely done, thanks @sanjayradadiya